### ISSUE TITLE
Raspberry Pi 4B crashes when running xmrig

### ISSUE THREAD

**Issue by GYKgamer** [xmrig/xmrig#3104]
When running command "xmrig" it would start as normal, but after a few seconds my raspberry pi will crash (see screenshot and video)

Running "xmrig --stress" crashes my raspberry pi
Running xmrig at one core doesn't crash but instead a segfault occurs
Running the app "Geekbench" (a CPU-intensive program) does not crash my raspberry pi
Trying to change the xmrig binary doesn't work either

Here is also where I got my help https://github.com/Botspot/pi-apps/issues/2096

https://user-images.githubusercontent.com/41442793/183134784-0207ac45-01a0-41e6-9aa4-d0cfc684a295.mp4

![image](https://user-images.githubusercontent.com/41442793/183134796-3daad81b-ed6b-4553-a67d-c2e2bcf648ec.png)

**Comment by SChernykh** [comment]
This is MoneroOcean version in your video, please test with the original xmrig. In any case, it shouldn't cause full system crash unless your Pi is unstable.

**Comment by GYKgamer** [comment]
I only use the MonerOcean pool? I don't know of any MonerOcean xmrig versions. All files come from this github page.

**Comment by SChernykh** [comment]
XMRig/6.18.0-mo1 is not a file from this github page, it's MoneroOcean fork.

**Comment by SChernykh** [comment]
Oh, I know why. You run your Pi in 32-bit mode, it is not supported on ARM. Install 64-bit OS and to mine properly.

**Comment by Botspot** [comment]
> Oh, I know why. You run your Pi in 32-bit mode, it is not supported by XMRig on ARM. Install 64-bit OS to mine properly.

@SChernykh, so you [fixed](https://github.com/xmrig/xmrig/issues/2895#issuecomment-1133113854) xmrig to run on armhf systems, but when users encounter problems on it, you say armhf is not supported?

**Comment by GYKgamer** [comment]
Oh hi @Botspot , Thanks for reminding me that this post existed. Now whenever I run "xmrig" (arm64) I get a segmentation Fault
![image](https://user-images.githubusercontent.com/41442793/187059422-8e2654a8-ba63-45a0-96a7-42b15f9b207a.png)

**Comment by SChernykh** [comment]
@Botspot it is not supported because we never test it on 32-bit ARM. That one time I had to install 32-bit OS on my RPi to fix this crash, and then I had to install back my regular 64-bit OS. I spen half of the day doing it. Also, RandomX doesn't have JIT for 32-bit ARM so it will be very very slow (less than 1 h/s)

@GYKgamer this is MoneroOcean version, ask in their github. If you manage to get this crash in the official XMRig version, post it in a new issue.
